### PR TITLE
document how to build a Feature from a WPK

### PR DESCRIPTION
**Pull request by denisbedin**
https://docs.microej.com/en/feature-buildfeature_from_wpk/

### CODE REVIEW

File: SDK6UserGuide/buildFeature.rst [hunk 1]
```diff
@@ -59,6 +59,29 @@ in the ``microej`` configuration block of the Gradle build file of the project::
...
+   dependencies {
+      microejApplication("com.mycompany:myapp:1.0.0")
+   }
+
+- A Multi-Sandbox Kernel must be defined.
+  Refer to the :ref:`sdk_6_select_kernel` page to learn how to provide a Kernel for a module project.
+
+Once these prerequisites are fulfilled, the Feature file of the dependent Application can be built::
+
+   ./gradlew buildFeatureFromWPK
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/buildFeature.rst:81]: Shouldn't we have the same tabs than in the section above for the call to the `buildFeature` task?

File: SDK6UserGuide/buildFeature.rst [hunk 2]
```diff
@@ -59,6 +59,29 @@ in the ``microej`` configuration block of the Gradle build file of the project::
...
+-------------------------------------------------
+
+The SDK provides the Gradle ``buildFeatureFromWPK`` task which allows to build a Feature file from a dependent Application.
+The prerequisites to use this task are:
+
+- An Application must be defined as dependency in the build file of the project with the ``microejApplication`` configuration::
+
+   dependencies {
+      microejApplication("com.mycompany:myapp:1.0.0")
+   }
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/buildFeature.rst:74]: Can the dependency be a WPK file directly?

File: SDK6UserGuide/buildFeature.rst (hunk 1 above)
Comment (denisbedin) [reply SDK6UserGuide/buildFeature.rst]: Done.

File: SDK6UserGuide/buildFeature.rst (hunk 2 above)
Comment (denisbedin) [reply SDK6UserGuide/buildFeature.rst]: Yes, I documented it.

File: SDK6UserGuide/buildFeature.rst [hunk 3]
```diff
@@ -59,6 +59,73 @@ in the ``microej`` configuration block of the Gradle build file of the project::
...
+
+Build a Feature file from a WPK
+-------------------------------
+
+The SDK provides the Gradle ``buildFeatureFromWPK`` task which allows to build a Feature file from the WPK of an Application.
+The prerequisites to use this task are:
+
+- The WPK must be defined:
+
+  - When the WPK is published in an artifact repository, you can use it by declaring a Module dependency::
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/buildFeature.rst:72]:
```suggestion
  - When the WPK is published as an artifact repository, you can use it by declaring a Module dependency::
```

Comment (denisbedin) [reply SDK6UserGuide/buildFeature.rst:72]: Isn't ` published in an artifact repository` better here ? The WPK is not a repository.

Comment (tdelhomenie-microej) [reply SDK6UserGuide/buildFeature.rst:72]: Right.

**Review by tdelhomenie-microej — APPROVED** [review]

**Pull request merged**